sql/_13_issues/_22_2h/cases/cbrd_24424.sql

```diff
@@ -0,0 +1,33 @@
+show heap header of dba._db_class;
```

Comment:
The test case does not contain the comments I made.

If this test case fails during development, we need to check what the test case is for.
If possible, please include comments as well.

Reply:
Okay. Added comments.